util.js

```diff
@@ -113,6 +208,7 @@ exports.debuglog = function(set) {
   return debugs[set];
 };
 
+var customInspectSymbol = typeof Symbol !== 'undefined' ? Symbol('util.inspect.custom') : undefined;
```

Comment:
**Node&nbsp;10.12** and&nbsp;newer&nbsp;registers [the&nbsp;`util.inspect.custom`&nbsp;symbol](https://nodejs.org/api/util.html#util_util_inspect_custom) as&nbsp;`nodejs.util.inspect.custom`, making&nbsp;it&nbsp;possible to&nbsp;use&nbsp;`Symbol.for('nodejs.util.inspect.custom')` to&nbsp;get&nbsp;the&nbsp;equivalent `util.inspect.custom`&nbsp;symbol without&nbsp;depending&nbsp;on&nbsp;`util`:

```suggestion
var customInspectSymbol = typeof Symbol !== 'undefined' && typeof Symbol.for !== 'undefined' ? Symbol.for('nodejs.util.inspect.custom') : undefined;
```

Reply:
Browsers, however, don’t register it, so I’d you don’t have util.inspect, there’s no point in having the symbol - why not just depend on util?

Reply:
Also, node 6.6-10.11 all required his code, and can’t rely on the shared symbol.